sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,3 +182,13 @@ def test_connectivity(seed=36):
     assert_equal(_graph_is_connected(graph), True)
     assert_equal(_graph_is_connected(csr_matrix(graph)), True)
     assert_equal(_graph_is_connected(csc_matrix(graph)), True)
+
+
+def test_spectral_embedding_deterministic():
+    """Test that Spectral Embedding is deterministic"""
```

Comment:
Could you pl. convert this to a comment? [ Related to #4250 ] ( also do the same for the test for `eigen_sign_flip` )